src/main/java/uk/gov/digital/ho/hocs/aws/MigrationLocalStackConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.digital.ho.hocs.aws;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.sqs.AmazonSQSAsync;
+import com.amazonaws.services.sqs.AmazonSQSAsyncClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.aws.messaging.config.annotation.EnableSqs;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+
+@EnableSqs
+@Configuration
+@Profile({"migration-local"})
```

Comment:
`migration` would be simpler here, you can run `local` to give you the SQS and then `migration` for the S3 as required.

Reply:
Fixed